client/src/lib/slate-pointers/ShowExpandedPointer.tsx

```diff
@@ -57,29 +57,40 @@ export class ShowExpandedPointer extends React.Component<any, any> {
   }
 
   public render() {
+    const isNestedAsExportingPointer = this.props.availablePointers.some(pointer => {
```

Comment:
This seems like it might become expensive for larger trees. For each expanded pointer, each time we render, we traverse (up to) all available pointers (which can be very many given our setup), and for each of them, we traverse (up to) all nodes. Is there some other way to achieve what this is doing? Or is there reason to think that it's not going to affect rendering speed much?

Reply:
I agree it doesn't look too scalable. Given the current approach to pointers, we actually don't really need it. So for now we can just take it out entirely. If, in the future, in this component, we want to know whether a pointer is nested, we can revisit the issue of how to most efficiently to determine this.

Reply:
https://github.com/oughtinc/mosaic/pull/145/commits/1677f11d0f392b823d4030f19d6db22a9de25816